### ISSUE TITLE
Custom styles for select element

### ISSUE THREAD

**Issue by btzr-io** [btzr-io/Villain#139]
Add custom styles for the select element component on the demo example.
> Note: it should be simple ( minimalist ) and not affect usability or accessibility.
